### PR TITLE
fixed currency filter error

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#2259

*Description of changes:*
- fixed filter not working in ternary expression
- updated version

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: vue/sbc-common-components/src/services/token.services.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ class TokenServices {
     })
   }
 
-  scheduleRefreshTimer (refreshEarlyTime :number = 0) {
+  scheduleRefreshTimer (refreshEarlyTime: number = 0) {
```
Comment (saravankumarpa) [inline vue/sbc-common-components/src/services/token.services.ts]: did u intend to commit this as well?  the token changes

Comment (severinbeauvais) [reply vue/sbc-common-components/src/services/token.services.ts]: Yes, cleanup.

File: vue/sbc-common-components/src/services/token.services.ts [hunk 2]
```diff
@@ -67,11 +67,12 @@ class TokenServices {
     clearTimeout(this.timerId)
   }
 
-  private scheduleRefreshToken (refreshEarlyTimeinMilliseconds:number) {
+  private scheduleRefreshToken (refreshEarlyTimeinMilliseconds: number) {
     let self = this
 
     // check if refresh token is still valid . Or else clear all timers and throw errors
-    let refreshTokenExpiresIn = self.kc.refreshTokenParsed['exp'] - Math.ceil(new Date().getTime() / 1000) + self.kc.timeSkew
```
Comment (severinbeauvais) [inline vue/sbc-common-components/src/services/token.services.ts]: FYI: this line is too long and causes a lint error

File: vue/sbc-common-components/src/services/token.services.ts (hunk 1 above)
Comment (saravankumarpa) [reply vue/sbc-common-components/src/services/token.services.ts]: ok..

Comment (saravankumarpa) [reply vue/sbc-common-components/src/services/token.services.ts]: I made these changes [not the linting one] in the other PR..so might be a conflict

Comment (severinbeauvais) [reply vue/sbc-common-components/src/services/token.services.ts]: I reverted these changes.

**Pull request merged**